Frontend/ui-driver/src/Screens/ReferralScreen/View.purs

```diff
@@ -295,7 +295,7 @@ noDataView state =
             [ width MATCH_PARENT
             , height WRAP_CONTENT
             , margin (Margin 24 0 24 30)
-            , text (getString PLEASE_WAIT_WHILE_WE_UPDATE_THE_DETAILS)
+            , text (getStringEnToHi PLEASE_WAIT_WHILE_WE_UPDATE_THE_DETAILS)
```

Comment:
why are this strings added as EnToHi?

Reply:
we have this workflow working for all languages

Reply:
this was the requirement to show in hindi only if merchant is yatrisathi cc: @aryan-13